### DISCUSSION TITLE
Cannot add the app on Obtainium

### DISCUSSION THREAD

**Discussion by xfzv** [breezy-weather/breezy-weather#83] [Q&A] [answered]
From the README:

https://github.com/breezy-weather/breezy-weather/blob/ab0257cbac3b73d2adeb8ae0061fe45eb3fbc186/README.md?plain=1#L24-L28

In Obtainium:

1. When trying to add the repo URL, I'm getting "Could not find a suitable release" message.
![2](https://github.com/breezy-weather/breezy-weather/assets/78810647/4e4df02e-e4b8-4620-93ea-bc6e15d47ed7)

2. The repo doesn't appear in search results. 
![1](https://github.com/breezy-weather/breezy-weather/assets/78810647/e0460944-3ee0-45e0-b2c9-8e52ec480eb9)

Am I missing something?

Thanks.

**Comment by papjul** [comment, marked as answer]
Enable "Include prereleases", Breezy Weather is still alpha and has no "stable" releases at the moment.

**Reply by xfzv** [reply]
Ah, that was it. Silly me.

Thank you!
